terraform/shared/shared.tf.json.template.py

```diff
@@ -344,7 +344,7 @@ def conformance_pack(name: str) -> str:
                 }
             },
             'trail': {
-                'bucket': '${aws_s3_bucket.logs.id}',
+                'bucket': '${aws_s3_bucket.trail.id}',
```

Comment:
This should have been a comprehension in the first place. Whenever you copy and paste code, you should consider alternatives. Copying code is almost always a problematic practice and in this case we can plainly see why. It costs the team a disappointing amount of time to catch and squash these types of bugs that could be prevented so easily.